web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1553,6 +1553,16 @@ export const formatters: Formatters = {
     format: ({ name, source, user }) =>
       `Okta assignment [${name}], source [${source}], user [${user}] cleanup has failed`,
   },
+  [eventCodes.OKTA_ACCESS_LIST_SYNC]: {
+    type: 'okta.access_list.sync',
+    desc: 'Okta access list synchronization completed',
+    format: () => `Okta access list synchronization successfully completed`,
```

Comment:
I'm not familiar with the details here. Is this syncing a single access list or multiple? If single, can we include the name of the list here?

Reply:
We're syncing multiple (maybe hundreds).